apps/site/lib/site_web/templates/customer_support/_form.html.eex

```diff
@@ -50,6 +50,18 @@
   <section>
     <h3>Additional Details (optional)</h3>
     <div>Adding more details helps us more effectively respond to your concerns.</div>
+     <div class="ada-container form-group">
+      <%= SiteWeb.PartialView.render("_checkbox.html", %{
+        form: f,
+        field: :ada_response,
```

Comment:
Was following the convention of `no_request_response`. Was interpreting as the responses from the feedback provider instead of response from MBTA. `ada_complaint` makes more sense in the  latter case.